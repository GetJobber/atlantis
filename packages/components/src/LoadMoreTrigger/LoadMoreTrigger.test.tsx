import React from "react";
import { render } from "@testing-library/react";
import { LoadMoreTrigger } from ".";

it("renders a LoadMoreTrigger", () => {
  const { container } = render(<LoadMoreTrigger onLoadMore={jest.fn()} />);
  expect(container).toMatchSnapshot();
});
