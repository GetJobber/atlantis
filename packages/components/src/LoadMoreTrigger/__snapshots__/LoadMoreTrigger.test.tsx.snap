// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a LoadMoreTrigger 1`] = `
<div>
  <div
    class="trigger"
    data-testid="ATL-LoadMore-Trigger"
  />
</div>
`;
